vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -61,7 +61,23 @@ def process_collection_item(item, key)
 
     def parse_node(node)
       new_result = parse_base_item(node)
-      # TODO, add more properties such as resources
+
+      new_result.merge!(
+        :identity_infra   => node.spec.send(:externalID) || nil,
```

Comment:
- Don't call .send with a symbol.  It's roughly the same as just calling `.externalID` directly, and this latter is preferred.
- `|| nil` seems redundant, or does `externalID` return `false` when it's not available?